streams/src/test/java/org/apache/kafka/streams/integration/IQv2StoreIntegrationTest.java

```diff
@@ -776,64 +779,55 @@ public void verifyStore() {
                 globalShouldRejectAllQueries();
             } else {
                 shouldRejectUnknownQuery();
-                shouldCollectExecutionInfo();
+//                shouldCollectExecutionInfo();
                 shouldCollectExecutionInfoUnderFailure();
                 final String kind = this.kind;
                 if (storeToTest.keyValue()) {
                     if (storeToTest.timestamped()) {
-                        final Function<ValueAndTimestamp<Integer>, Integer> valueExtractor =
-                            ValueAndTimestamp::value;
-                        if (kind.equals("DSL")) {
-                            shouldHandleKeyQuery(2, valueExtractor, 5);
-                            shouldHandleRangeDSLQueries(valueExtractor);
-                        } else {
-                            shouldHandleKeyQuery(2, valueExtractor, 5);
-                            shouldHandleRangePAPIQueries(valueExtractor);
-                        }
-                    } else {
-                        final Function<Integer, Integer> valueExtractor = Function.identity();
-                        if (kind.equals("DSL")) {
-                            shouldHandleKeyQuery(2, valueExtractor, 5);
-                            shouldHandleRangeDSLQueries(valueExtractor);
-                        } else {
-                            shouldHandleKeyQuery(2, valueExtractor, 5);
-                            shouldHandleRangePAPIQueries(valueExtractor);
-                        }
-                    }
-                }
+                        shouldHandleKeyQuery(2,  5);
+                        shouldHandleTimestampedKeyQuery(2, 5);
+                        shouldHandleRangeQueries();
+                        shouldHandleTimestampRangeQueries();
+                  }
+                    else {
+                        shouldHandleKeyQuery(2, 5);
```

Comment:
ROCK_KV DSL store were execute the code in this else statement.

Reply:
For ROCK_KV DSL store, and this time Cache is true, the data we get is not through the `KeyValueToTimestampedKeyValueByteStoreAdapter `'s query